trp_project_scrum/security/ir.model.access.csv

```diff
@@ -2,7 +2,6 @@ id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 collaborators_followers_read_project,collaborators_followers_read_project,project.model_project_project,collaborators_followers,1,0,0,0
 collaborators_followers_read_write_project_task,collaborators_followers_read_project_task,project.model_project_task,collaborators_followers,1,1,1,0
 collaborators_followers_allow_read_project_task_type,collaborators_followers_allow_read_project_task_type,project.model_project_task_type,collaborators_followers,1,0,0,0
-collaborators_followers_cr_attachments,collaborators_followers_cr_attachments,base.model_ir_attachment,collaborators_followers,1,0,1,0
```

Comment:
please read how permissions work in Odoo, you need an ACL for some operation to be allowed in the first place, and on top of that you add a rule that restrict this global permission to the records you want to permit to the specific group